instructure/canvas-android
# [MBL-16138][Student][Teacher] Share Extension basic UI + File Upload

refs: MBL-16138
affects: Student, Teacher
release note: none

test plan: 

1. Check the Share Extension functionality when sharing into the app.
2. Smoke test the File Upload dialog

- User files in both apps
- Discussion reply in both apps
- Create discussion in both apps
- Messages in both apps
- Create/Edit announcement in the Teacher app
